content/articles/cross-site-request-forgery-attack/index.md

```diff
@@ -0,0 +1,126 @@
+Cross-site request forgery (CSRF) is the third massive security vulnerability in web applications from [Cross Site Scripting (XSS)](https://www.section.io/engineering-education/how-to-prevent-cross-site-scripting-in-node-js/) and [SQL injection](https://www.section.io/engineering-education/how-to-fix-and-prevent-sql-injection-in-wordpress/). Web browsers are the most affected. When given a file to run, they can't tell if it is malicious or not. So, the web browsers go ahead and run the files.
+
```

Comment:
Please provide links that explain cross site scripting and SQL injections at a general level. You can then add the section article links at the bottom of your article under a further reading section.